README.md

```diff
@@ -28,7 +28,7 @@ This repository hosts the CSI KubeVirt driver and all of its build and dependent
 - Enable HotplugVolumes feature gate
     - In case your Kubevirt namespace has the ConfigMap 'kubevirt-config' then use `deploy/example/kubevirt-config.yaml` for adding the feature gate to it. Look at the path {.data.feature-gates}
     - Otherwise, add the feature gate to the resource of type Kubevirt. There should be a single resource of this type and its name is irrelevant. See `deploy/example/kubevirt.yaml`
-    - Pay attention that in some deployments there are operators that will restore previous configuration. You will have to stop these operators for editing the resources. E.g. hco-operator in HCO.
+    - Pay attention that in some deployments there are operators that will restore previous configuration. You will have to stop these operators for editing the resources. Some operators allow configuration through their own CRD. HCO is such an operator. See `deploy/example/hco-cr.yaml` to understand how HCO feature gates are configured.
 
```

Comment:
I wanted to add this oneliner according to [this](https://github.com/kubernetes/kubernetes/issues/58414) CRD patching isn't supported.
`oc patch hyperconverged kubevirt-hyperconverged -p '{"spec": {"featureGates": {"HotplugVolumes": true}}}'`

/lgtm
/approve